src/core/localiface.py

```diff
@@ -65,6 +65,21 @@
             raise IOError('Not a directory')
     except IOError as e:
         raise IOError(f'Could not stat storagehomepath folder {homepath}: {e}')
+    # all right but inform the user
+    log.warning('msg="Use this local storage interface for test/development purposes only, not for production"')
+
+def healthcheck():
```

Comment:
Maybe you can set the result string and return it in a `finally` to avoid this complain from CodeQL.

https://github.com/cs3org/wopiserver/pull/122/files#diff-25c605a5dcb9bcb40d0685b5aa2f93d64522c786754ee475f736d46844478413R52